### DISCUSSION TITLE
Run getting killed after 1 epoch

### DISCUSSION THREAD

**Discussion by nighting0le01** [yoshitomo-matsubara/torchdistill#380] [Q&A] [answered]
![image](https://github.com/yoshitomo-matsubara/torchdistill/assets/81643693/b535c74a-80fb-4be3-ba0a-9c429cd7ba66)
![image](https://github.com/yoshitomo-matsubara/torchdistill/assets/81643693/01b1e233-c308-46b8-b3ef-8e275c726ac0)

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Use `ckpt` instead of `dst_ckpt` for scripts with torchdistill v.0.3.3. I've already asked you to share texts instead of screenshots https://github.com/yoshitomo-matsubara/torchdistill/discussions/378#discussioncomment-6536059
